packages/framework/collabspaces/src/collabSpaces.ts

```diff
@@ -447,23 +457,42 @@ export class TempCollabSpaceRuntime
 
 	private cellChanged(row: number, col: number) {
 		const info = this.getCellInfo(row, col);
-		if (info.value !== undefined && isChannelDeffered(info.channelInfo?.type)) {
-			const channelId = info.channelId;
-			// Need to update channel and convert it to real thing.
-			const deferredChannel = this.deferredChannels.get(channelId);
-			assert(deferredChannel !== undefined, "deferred channel not found");
-			this.deferredChannels.delete(channelId);
 
-			this.destroyChannelCore(channelId);
+		if (info.value !== undefined) {
+			if (col === -1 || row === -1) {
```

Comment:
This is more of my mistake but would be great if you could fix it.
Calling getCellInfo() above with either row or col being -1 is not great. It's great that it does not crash today, but it could, and it also does a bunch of unnecessary work in such case.
I'd advice to actually not even get into here with -1, but make a fork much earlier - in colsChanged callback and have separate path for -1 that would not call this.getCellInfo, but would call minimum APIs (I assume go directly against this.matrix.getCell)